src/common/application-event.ts

```diff
@@ -27,7 +27,7 @@ export interface ApplicationEvents {
    */
   instance: (event: InstanceEvent) => void
   /**
-   * Broadcast instance to inform other applications nodes in cluster about its existence
+   * Broadcast instance to inform other application nodes in cluster about it's existence
```

Comment:
`applications` as other nodes connecting via websocket. There can be multiple ones.
I believe it is `its`

Reply:
Changed to "bridge", fixed instances of "it's" being contracted, not possessive